controllers/discovery/initDiscoveryDomainData.js

```diff
@@ -26,7 +32,7 @@ const InitDiscoveryDomainData = async (req, res) => {
                 AND A.user_id_follower = :userId
             RIGHT JOIN domain_page C 
                 ON C.domain_page_id = A.domain_id_followed
-            WHERE C.credder_score >= ${CREDDER_MIN_SCORE} AND C.status = true
+            WHERE C.credder_score >= :credderMinScore AND C.status = true
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

Good job on parameterizing the `credderMinScore` value. This enhances security by preventing SQL injection attacks.



<!-- This is an auto-generated comment by OSS CodeRabbit -->